docs/resources.md

```diff
@@ -550,6 +550,7 @@ The Cluster resource has the following parameters:
     certificate.
 -   `cadata` (required): holds PEM-encoded bytes (typically read from a root
     certificates bundle).
+-   `authProvider`: specifies an optional kubectl auth provider to use.
```

Comment:
Can you document some options? Or link to where I can find the options?

Reply:
I tried. There are literally no docs on this...

Reply:
Not really documentation, but this is the list of known plugins afaik: https://github.com/kubernetes/client-go/blob/master/plugin/pkg/client/auth/plugins.go#L20